openqa-prepare-mm-setup

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env perl
+# Copyright 2020 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+use strict;
+
+system("zypper in -y os-autoinst-openvswitch");
+system("systemctl enable --now os-autoinst-openvswitch");
+
+my $bridge = 1;
+my $openvswitch = "/etc/sysconfig/os-autoinst-openvswitch";
+open(file, '>', $openvswitch) or die "Failed to open $openvswitch for writing!";
+print file "OS_AUTOINST_USE_BRIDGE=br$bridge";
+close(file);
+
+system("ovs-vsctl add-br br$bridge");
+print("Added bridge $bridge");
+
+my $bridge_file = "/etc/sysconfig/network/ifcfg-br$bridge";
+my $ip = "10.0.2.2/15";
+my $config = "
+BOOTPROTO='static'                                                                                                         
+IPADDR='$ip'                                                                                                       
+STARTMODE='auto'                                                                                                           
+ZONE=trusted                                                                                                               
+OVS_BRIDGE='yes'                                                                                                           
+PRE_UP_SCRIPT=\"wicked:gre_tunnel_preup.sh\"
+";
+
+for (my $i = 0; $i < 148; $i++) {
+    $config .= "OVS_BRIDGE_PORT_DEVICE_$i='tap$i'\n";
+
+    my $tapconfig = "
+    BOOTPROTO='none'
+    IPADDR=''
+    NETMASK=''
+    PREFIXLEN=''
+    STARTMODE='auto'
+    TUNNEL='tap'
+    TUNNEL_SET_GROUP='nogroup'
+    TUNNEL_SET_OWNER='_openqa-worker'
+    ZONE=trusted
+    ";
+
+    my $tap_file = "/etc/sysconfig/network/ifcfg-tap$i";
+    open(file, '>', $tap_file) or die "Failed to open $tap_file for writing!";
+    print file $tapconfig;
+    close(file);
+}
+
+open(file, '>', $bridge_file) or die "Failed to open $bridge_file for writing!";
+print file $config;
+close(file);
+print("Config written!");
+
+my $gre = "/etc/wicked/scripts/gre_tunnel_preup.sh";
+open(file, '>', $gre) or die "Failed to open $gre for writing!";
+
+my $gre_config = "
```

Comment:
a Here-document should be used to avoid having to escape the `"` abd `$`

Reply:
Single quotes should also be fine here actually